docs/tool/GUI.ModsManager/FAQ.md

```diff
@@ -0,0 +1,55 @@
+# OpenKH Mod Manager FAQs
+
+### Return to the [Main Page](index.md)
+
+### Special Thanks to:
+* Shananas for the continued upkeep of the OpenKH Mod Manager when something new arises
+* MainMemory for her updating of Panacea to work with the PC releases
+* Xeeynamo for the initial rough version of the program
+* Alios for discovering a major issue with the Mod Manager so that I may include an answer to the problem in this FAQs document
+* Everyone else in the OpenKH community for helping make the modding environment for these games as welcoming and exciting as it has been
+
+## [Table of Contents]()
+* ### [Is extracting my games mandatory for the mod manager to work?](#q-is-extracting-my-games-mandatory-for-the-mod-manager-to-work)
+* ### [Lua mods aren't working!](#q-lua-mods-arent-working)
+* ### [Does OpenKH support Steam Deck/Linux?](#q-does-openkh-support-steam-decklinux)
+* ### [I moved my OpenKH install and now I can't use the mod manager!](#q-i-moved-my-openkh-install-and-now-i-cant-use-the-mod-manager)
+* ### [I can't open the mod manager on my removeable storage device!]()
+* ### [I want to start making mods but don't know how](#q-i-want-to-start-making-mods-but-dont-know-how)
+
+---
+
+### Q: Is extracting my games mandatory for the mod manager to work?
+
+> **A:** For the best user experience, yes. We understand it takes up a lot of storage, but it allows you to retain your unmodified game files through a dynamic file replacement system instead of relying on a slow and tedious patching method. Some tips to reduce the storage used by game extraction are:
+> * Only extract the game(s) you are actively playing/modding.
+> * Enable `Dev View` by going to `Info > Dev View` inside the mod manager. This will allow you to patch your game(s) instead.
+    >   * **WARNING:** This **<ins>will</ins>** make helping debug problems with your game more difficult, and mod making far more arduous. 
+    >   * To revert your game to a completely vanilla slate, you may attempt to use the `Full Restore` options under `Patching > Full Restore`, but we cannot guarantee it will work if you have not properly kept a backup of your `.pkg` and `.hed` files. If this is the case, you may need to delete the respective game's `.pkg` and `.hed` files and verify the integrity of your game through Steam or EGS.
+---
+### Q: Lua mods aren't working!
+
+> **A:** Did you download and place either `Lua Backend` or `Lua Frontend` where your games are installed? If not, OpenKH can do this for you through the mod manager initialization wizard. If you need to configure `Lua Backend`, go to `Settings > Run wizard` until you reach the point where you can install `Lua Backend`. Select your games and click the button to configure it.

```

Comment:
I'm not sure if it needs to be changed but Mod Manager specifically downloads Lua Backend for use with panacea. If the user does not install panacea Lua Backend will not load. IF the user manually installs Lua Backend without renaming it aka leaving it as `DBGHELP.dll` instead of renaming it to `LuaBackend.dll` mod manager will see it as not installed.
Basically as the main intention of mod manager is a mod loader using panacea the install + configuration of Lua Backend is specifically desgined for use with panacea. This can be changed if desired (it would require me to use some kind of hash to make sure its not another dll named `DBGHELP.dll`)so honestly probably not worth it